cmd/dimension.go

```diff
@@ -63,20 +64,26 @@ var listDimensionsCmd = &cobra.Command{
 	Example: "corectl dimension ls",
 
 	Run: func(ccmd *cobra.Command, args []string) {
-		listEntities(ccmd, args, "dimension", !viper.GetBool("bash"))
+		state := internal.PrepareEngineState(rootCtx, headers, false)
+		items := internal.ListDimensions(state.Ctx, state.Doc)
+		if viper.GetBool("bash") {
```

Comment:
In my opinion we check the consistency here. For other printers I think we fetch the flags in the printer to now which format to print. Here we pass it to the printer. I am fine either way as long as we are consistent.